recipes/libiconv/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+    name: libiconv
+    version: 1.14
+
+source:
+    fn: libiconv-1.14.tar.gz
+    url: http://ftp.gnu.org/pub/gnu/libiconv/libiconv-1.14.tar.gz
+    md5: e34509b1623cec449dfeb73d7ce9c6c6
+    patches:
+        - patch1.patch  # [linux]
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - gcc
```

Comment:
Trying to drop this in PR ( https://github.com/conda-forge/libiconv-feedstock/pull/2 ).